src/main/java/com/google/devtools/build/lib/remote/options/RemoteOptions.java

```diff
@@ -483,6 +483,28 @@ public RemoteOutputsStrategyConverter() {
               + "that loads objects from the CAS on demand.")
   public String remoteDownloadSymlinkTemplate;
 
+  @Option(
+      name = "remote_output_service",
+      defaultValue = "null",
+      documentationCategory = OptionDocumentationCategory.REMOTE,
+      effectTags = {OptionEffectTag.UNKNOWN},
+      help =
+          "A URI of a remote output service. A remote output service is a daemon that runs next "
+              + "to Bazel, managing the contents of the bazel-out/ directory. It may use systems "
+              + "like FUSE to add features such as snapshotting and lazy loading of objects stored "
+              + "in a remote Content Addressable Storage.")
+  public String remoteOutputService;
+
+  @Option(
+      name = "remote_output_service_output_path_prefix",
+      defaultValue = "null",
+      documentationCategory = OptionDocumentationCategory.REMOTE,
+      effectTags = {OptionEffectTag.UNKNOWN},
+      help =
+          "The path at which all output path directories are visible that are managed by the "
+              + "remote output service.")
```

Comment:
Responding bottom up:

The reason why this must be provided on the Bazel side is because this information may simply not be known on the Remote Output Service side. You may run bb_clientd in container A, and instances of Bazel in containers B to Z. The FUSE file system that is created by bb_clientd may not be placed at the same absolute path as seen from Bazel's perspective.

This flag is indeed required when `--remote_output_service` is provided. Let me put an explicit check for this in place.

Reply:
--remote_output_service without --remote_output_service_output_path_prefix will now fail.